frontend/test/metabase/scenarios/dashboard/title-drill.cy.spec.js

```diff
@@ -48,39 +65,37 @@ describe("scenarios > dashboard > title drill", () => {
         },
       },
       display: "scalar",
-    }).then(({ body: { id: card_id } }) => {
-      cy.createDashboard("16181D").then(({ body: { id: dashboard_id } }) => {
-        // Add previously created question to the dashboard
-        cy.request("POST", `/api/dashboard/${dashboard_id}/cards`, {
-          cardId: card_id,
-        }).then(({ body: { id } }) => {
-          cy.addFilterToDashboard({ filter, dashboard_id });
-
-          cy.request("PUT", `/api/dashboard/${dashboard_id}/cards`, {
-            cards: [
-              {
-                id,
-                card_id,
-                row: 0,
-                col: 0,
-                sizeX: 8,
-                sizeY: 6,
-                parameter_mappings: [
-                  {
-                    parameter_id: filter.id,
-                    card_id,
-                    target: ["dimension", ["template-tag", "filter"]],
-                  },
-                ],
-              },
-            ],
-          });
+    };
+
+    cy.createNativeQuestionAndDashboard({ nativeQuestionDetails }).then(
+      ({ body: { id, card_id, dashboard_id } }) => {
+        cy.addFilterToDashboard({ filter, dashboard_id });
+
+        // Connect filter to the card
+        cy.request("PUT", `/api/dashboard/${dashboard_id}/cards`, {
+          cards: [
+            {
+              id,
+              card_id,
+              row: 0,
+              col: 0,
+              sizeX: 8,
+              sizeY: 6,
+              parameter_mappings: [
+                {
+                  parameter_id: filter.id,
+                  card_id,
+                  target: ["dimension", ["template-tag", "filter"]],
+                },
+              ],
+            },
+          ],
```

Comment:
This is the next thing I'll try to tackle.
It's unnecessarily long and repetitive to write/read. Some helper function can solve this.

Conceptually, something like this.
```javascript
editDashboardCard({oldCard, alteredProperties, dashboard_id});

//under the hood, this is a simple
Object.assign({}, oldCard, alteredProperties)

// so we can have something like
connectFilterToCard({oldCard, alteredProperties: parameter_mappings, dashboard_id})

// btw `createNativeQuestionAndDashboard` already gives us exactly what we need
// its `response.body` presents the existing/ oldCard
cy.createNativeQuestionAndDashboard({ nativeQuestionDetails }).then(
({ body: oldCard }) => {
    const { id, card_id, dashboard_id } = oldCard;
    
    // go about your business...
  });
});
```